### DISCUSSION TITLE
debian11

### DISCUSSION THREAD

**Discussion by KOF136** [bohanwood/debi#99] [General]
使用默认参数dd --version 11 最新版的内核是5.10.0-23,dd完成开机后 是5.10.0-22使用系统更新也是提示没有升级包.
打开/etc/apt/sources.list 文件查看源发现security的源被注释掉,
# Line commented out by installer because it failed to verify:
#deb http://security.debian.org/debian-security bullseye/updates main
# Line commented out by installer because it failed to verify:
#deb-src http://security.debian.org/debian-security bullseye/updates main
去掉注释也不能更新,正确的源应该是 http://security.debian.org/debian-security bullseye-security main 正常更新到5.10.0-23内核
可以通过什么参数自定义这个源

**Comment by bohanwood** [comment]
非常抱歉，是更新 Debian 12 支持的时候弄错了！
最新版已经更新

**Reply by KOF136** [reply]
重新用最新版的脚本DD了一下 debian11已经正常更新最新内核了
